packages/traceloop-sdk/traceloop/sdk/evaluator/evaluator.py

```diff
@@ -0,0 +1,102 @@
+import os
+import httpx
+from typing import Dict, Optional
+
+from traceloop.sdk.version import __version__
+from .model import (
+    InputExtractor,
+    InputSchemaMapping,
+    ExecuteEvaluatorRequest,
+    ExecuteEvaluatorResponse,
+    ExecutionResponse,
+)
+from .stream_client import SSEClient
+
+
+class Evaluator:
+    """
+    Evaluator class for executing evaluators with SSE streaming
+    """
+
+    @classmethod
+    def _create_async_client(cls) -> httpx.AsyncClient:
+        """Create new async HTTP client"""
+        api_key = os.environ.get("TRACELOOP_API_KEY", "")
+        if not api_key:
+            raise ValueError("TRACELOOP_API_KEY environment variable is required")
+
+        return httpx.AsyncClient(
+            headers={
+                "Authorization": f"Bearer {api_key}",
+                "Content-Type": "application/json",
+                "User-Agent": f"traceloop-sdk/{__version__}",
+            },
+            timeout=httpx.Timeout(120.0),
+        )
+
+    @classmethod
+    async def run_experiment_evaluator(
+        cls,
+        evaluator_slug: str,
+        task_id: str,
+        experiment_id: str,
+        experiment_run_id: str,
+        input: Dict[str, str],
+        timeout_in_sec: int = 120,
+        evaluator_version: Optional[str] = None,
+        client: Optional[httpx.AsyncClient] = None,
+    ) -> ExecutionResponse:
+        """
+        Execute evaluator with input schema mapping and wait for result
+
+        Args:
+            evaluator_slug: Slug of the evaluator to execute
+            input: Dict mapping evaluator input field names to their values. {field_name: value, ...}
+            client: Shared HTTP client for connection reuse (optional)
+            context_data: Context data to be passed to the evaluator (optional)
+            evaluator_version: Version of the evaluator to execute (optional)
+            timeout_in_sec: Timeout in seconds for execution
+
+        Returns:
+            ExecutionResponse: The evaluation result from SSE stream
+        """
+        schema_mapping = InputSchemaMapping(
+            root={k: InputExtractor(source=v) for k, v in input.items()}
+        )
+        request = ExecuteEvaluatorRequest(
+            input_schema_mapping=schema_mapping,
+            evaluator_version=evaluator_version,
+            task_id=task_id,
+            experiment_id=experiment_id,
+            experiment_run_id=experiment_run_id,
+        )
+        api_endpoint = os.environ.get("TRACELOOP_BASE_URL", "https://api.traceloop.com")
```

Comment:
shouldn't be here - you should get the API endpoint from the http client when initializing the evaluator class (similar to other APIs - https://github.com/traceloop/openllmetry/blob/main/packages/traceloop-sdk/traceloop/sdk/annotation/base_annotation.py)